File: official/docs/go/billing/retrieve_payment_methods.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

	paymentMethods, err := client.RetrievePaymentMethods()

	fmt.Println(paymentMethods) 
}

File: official/docs/csharp/billing/retrieve-payment-methods.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Xunit;
using Xunit.Sdk;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples;

public class Examples
{
    [Fact]
    public async Task RetrievePaymentMethods()
    {
        string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

        EasyPost.ClientManager.SetCurrent(apiKey);

        PaymentMethod paymentMethods = await Billing.RetrievePaymentMethods();

        new TestOutputHelper().WriteLine(JsonConvert.SerializeObject(paymentMethods, Formatting.Indented));
    }
}
